packages/blockchain-wallet-v4-frontend/src/components/Form/TextInputWithClipboard/index.tsx

```diff
@@ -35,9 +35,9 @@ const TextInputWithClipboard: React.FC<Props> = ({ value }) => {
   return (
     <Wrapper>
       <StyledInput disabled value={value} />
-      <IconWrapper>
-        <CopyToClipboard color='blue400' value={value} />
-      </IconWrapper>
+      {/* <IconWrapper> */}
+      {/*  <CopyToClipboard color='blue400' value={value} /> */}
+      {/* </IconWrapper> */}
```

Comment:
@pedroapfilho @awiermanncasas-bc 
1) can one of you update this to use the copy for clipboard hook?